docker/Dockerfile.gaia

```diff
@@ -0,0 +1,74 @@
+FROM metabrainz/python:2.7
+
+# Installing dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+                       build-essential \
+                       libyaml-dev \
+                       libfftw3-dev \
+                       libavcodec-dev \
+                       libavformat-dev \
+                       python-numpy-dev \
+                       python-numpy \
+                       git \
+                       libsamplerate0-dev \
+                       libtag1-dev \
+                       libqt4-dev \
+                       swig2.0 \
+                       pkg-config \
+                       libpq-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /hl_extractor
+WORKDIR /hl_extractor
+
+COPY ./hl_extractor /hl_extractor/hl_extractor
+
+# PostgreSQL password
+ENV PGPASSWORD "acousticbrainz"
+
+# Gaia
+# See https://github.com/MTG/gaia
+RUN git clone https://github.com/MTG/gaia /tmp/gaia \
+    && cd /tmp/gaia \
+    && ./waf configure --with-python-bindings \
+    && ./waf \
+    && ./waf install
+
+# Essentia
+# See http://essentia.upf.edu/documentation/installing.html
+RUN git clone https://github.com/MTG/essentia /tmp/essentia \
+    && cd /tmp/essentia \
+    && ./waf configure --mode=release --with-gaia --with-example=streaming_extractor_music_svm \
+    && ./waf \
+    && ./waf install \
+    && ldconfig \
+    && cp /tmp/essentia/build/src/examples/essentia_streaming_extractor_music_svm /hl_extractor/hl_extractor/streaming_extractor_music_svm
+
+# SVM models
+RUN mkdir /tmp/models \
+    && cd /tmp/models \
+    && curl --silent -o models.tar.gz http://essentia.upf.edu/documentation/svm_models/essentia-extractor-svm_models-v2.1_beta1.tar.gz \
+    && tar -xvzf models.tar.gz \
+    && mv /tmp/models/v2.1_beta1/svm_models /hl_extractor/hl_extractor/
+
+# Cleanup
+RUN rm -rf /tmp/gaia
+RUN rm -rf /tmp/essentia
+
+# Python dependencies
+COPY requirements.txt /hl_extractor/requirements.txt
+RUN mkdir /hl_extractor/docs/
+COPY docs/requirements.txt /hl_extractor/docs/requirements.txt
+RUN pip install -r requirements.txt
+
+# Install Dockerize
+ENV DOCKERIZE_VERSION v0.2.0
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+COPY ./db /hl_extractor/db
+COPY ./default_config.py /hl_extractor/default_config.py
```

Comment:
Why is this needed?

Reply:
ah, I thought I cleaned up this, it was in the original Dockerfile but isn't needed

Reply:
in fact, this is the wrong file. I hope I didn't delete all of my changes